doc/controls/DrawerFlyoutPresenter.md

```diff
@@ -15,26 +15,71 @@ xmlns:utu="using:Uno.Toolkit.UI"
+
+    The workaround here is to define the template in the consuming assembly again.
+    It doesn't matter if it is used or not.
+-->
+<win:Style x:Key="MUX6388_Workaround_ForDefinitionOnly" TargetType="FlyoutPresenter">
+    <Setter Property="Template">
+        <Setter.Value>
+            <ControlTemplate TargetType="FlyoutPresenter">
+                <!-- This is not the full template, we are just making explicit reference to these definitions below -->
+                <utu:DrawerFlyoutPresenter OpenDirection="{TemplateBinding utu:DrawerFlyoutPresenter.OpenDirection}"
+                                            DrawerDepth="{TemplateBinding utu:DrawerFlyoutPresenter.DrawerDepth}"
+                                            LightDismissOverlayBackground="{TemplateBinding utu:DrawerFlyoutPresenter.LightDismissOverlayBackground}"
+                                            IsGestureEnabled="{TemplateBinding utu:DrawerFlyoutPresenter.IsGestureEnabled}" />
+            </ControlTemplate>
+        </Setter.Value>
+    </Setter>
+</win:Style>
+```
+
 ### Properties
 Property|Type|Description
 -|-|-
 OpenDirection|DrawerOpenDirection|Gets or sets the direction in which the drawer opens toward.<br/>note: The position of drawer when opened is the opposite of this value.
+DrawerDepth|GridLength|Get or sets the depth (width or height depending on the `OpenDirection`) of the drawer.\*
 LightDismissOverlayBackground|Brush|Gets or sets the brush used to paint the light dismiss overlay. The default value is `#80808080` (from the default style).
 IsGestureEnabled|bool|Get or sets a value that indicates whether the user can interact with the control using gesture. The default value is `true`.
 
 ## Usage
+- For DrawerDepth, this value has 3 mode based on `GridUnitType`:
+    ```xml
+    <Style TargetType="FlyoutPresenter">
+        <Setter Property="utu:DrawerFlyoutPresenter.DrawerDepth" Value="Auto" />
+        <Setter Property="utu:DrawerFlyoutPresenter.DrawerDepth" Value="0.66*" />
+        <Setter Property="utu:DrawerFlyoutPresenter.DrawerDepth" Value="150" />
+    <!-- and/or -->
+    <DrawerFlyoutPresenter DrawerDepth="Auto" />
+    <DrawerFlyoutPresenter DrawerDepth="0.66*" />
+    <DrawerFlyoutPresenter DrawerDepth="150" />
+    ```
+    - `GridUnitType.Auto`: Fit to flyout content.
+    - `GridUnitType.Star`: At given ratio of screen/flyout width or height. Valid range is between 0* and 1*, excluding 0* itself.
+    - `GridUnitType.Pixel`: Fixed at the given value.
 
 To use this, simply use a `Flyout` with `Placement="Full"` and one of the followings as the `FlyoutPresenterStyle`:
-> note: The direction here indicates the initial position of the drawer. The open direction is the opposite.
+> note: The direction here indicates the initial position of the drawer (where it opens from). The open animation direction is the opposite.
```

Comment:
Is this meant to show as an alert on DocFX? If so, we should set it as `[!NOTE]` on its own line

[docfx usage](https://dotnet.github.io/docfx/docs/markdown.html?tabs=linux%2Cdotnet#alerts)

Reply:
no, this is just a btw for clarification